frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/dev-console-create-vm-form.tsx

```diff
@@ -0,0 +1,229 @@
+  const templateParam = urlParams.get('common-template');
+  const namespace = urlParams.get('namespace') || 'default';
+
+  const {
+    pods,
+    dataVolumes,
+    pvcs,
+    userTemplates,
+    baseTemplates,
+    resourcesLoaded,
+    resourcesLoadError,
+  } = useVmTemplatesResources(namespace);
+
+  const selectedTemplate = filterTemplates(userTemplates, baseTemplates)
+    .filter((tmp) => {
+      const tempSourceStatus = getTemplateSourceStatus({
+        pods,
+        pvcs,
+        dataVolumes,
+        template: tmp.variants[0],
+      });
+
+      if (isTemplateSourceError(tempSourceStatus) || !tempSourceStatus?.isReady) {
+        return false;
+      }
+      return true;
+    })
+    .find((tmp) => templateParam === tmp?.metadata?.name);
+
+  const [state, dispatch] = React.useReducer(formReducer, initFormState(namespace));
+  const [vmCreated, setCreated] = React.useState(false);
+  const [isSubmitting, setSubmitting] = React.useState(false);
+  const [createError, setCreateError] = React.useState<string>();
+  const [createdVmUid, setCreatedVmUid] = React.useState<string>();
+  const [scConfigMap, scLoaded, scError] = useStorageClassConfigMap();
+  const template = selectedTemplate?.variants?.[0];
+  const sourceStatus = getTemplateSourceStatus({
+    pvcs,
+    pods,
+    dataVolumes,
+    template,
+  });
+
+  return (
+    <div className="co-m-pane__body">
+      <h1 className="co-m-pane__heading">Review and create virtual machine</h1>
+      <Helmet>
+        <title>Virtual Machines</title>
+      </Helmet>
+      <StatusBox
+        data={template}
+        loaded={resourcesLoaded}
+        loadError={resourcesLoadError}
+        label="Virtual Machine Template"
+        EmptyMsg={() => (
+          <EmptyState>
+            <Title headingLevel="h4" size="lg">
+              Error Loading Template
+            </Title>
+            <EmptyStateBody>
+              Virtual machine template: <strong>{templateParam}</strong> not found.
+            </EmptyStateBody>
+            <Button
+              variant="primary"
+              onClick={() => history.push(`/add/ns/${namespace}/virtualization`)}
+            >
+              Back to templates catalog
+            </Button>
+          </EmptyState>
+        )}
+      >
+        <div className="row">
+          {vmCreated ? (
+            <Bullseye>
+              <SuccessResultsComponent
+                name={state.name}
+                namespace={state.namespace}
+                detailsBtnText="View in topology"
+                detailsHref={`/topology/ns/${state.namespace}?view=graph&selectId=${createdVmUid}`}
+                successText="View virtual machine in topology to see the current status"
+                showListButton={false}
+              />
+            </Bullseye>
+          ) : (
+            template && (
+              <>
+                <div className="col-md-7 col-md-push-5 co-catalog-item-info">
```

Comment:
it would be better to use PF `Split` and `Stack` layout

Reply:
I was following `instantiate-template.tsx`, not sure its a good idea to change the layout

Reply:
ok, lets keep it as is